app/models/user.rb

```diff
@@ -1,3 +1,5 @@
 class User < ActiveRecord::Base
   include GinaAuthentication::UserModel
+
+  has_many :exports
```

Comment:
should also have dependent destroy, if user gets deleted it needs to clear their exports as well.